### PR TITLE
[patch] minor changes - path update 

### PR DESCRIPTION
**Pull request by padmankosalaram**
- [ ] update to dir path /tmp
- [ ] oc login check and exit 1 or 0 based on provision / deprovision flow
- [ ] vpc check exit 0 in deprovision flow

### CODE REVIEW

File: tekton/src/tasks/gitops/gitops-deprovision-suite-config.yml.j2 [hunk 1]
```diff
@@ -69,7 +69,11 @@ spec:
         export OCP_USERNAME=$(cat $ROSA_CONFIG | yq -r '.data.username')
         export ROSA_CLUSTER_ADMIN_PASSWORD=$(cat $ROSA_CONFIG | yq -r '.data.admin_password')
         oc login -u $OCP_USERNAME -p $ROSA_CLUSTER_ADMIN_PASSWORD $OCP_SERVER --insecure-skip-tls-verify
-
+        OC_RC=$?
+        if [ $OC_RC -ne 0 ]; then
+          echo "oc login failed, exiting with error.."
+          exit 1
+        fi
```
Comment (whitfiea) [inline tekton/src/tasks/gitops/gitops-deprovision-suite-config.yml.j2:76]: I think we have to be careful here. The cluster might not exist but we still want to carry on and remove the pieces from github. This then links to the other piece of work which is to allow deprovsion to run so we need to timeout the argo sync check. So I think we should log the error here but carry on.

File: tekton/src/tasks/gitops/gitops-deprovision-suite-workspace.yml.j2 [hunk 2]
```diff
@@ -66,7 +66,11 @@ spec:
         export OCP_USERNAME=$(cat $ROSA_CONFIG | yq -r '.data.username')
         export ROSA_CLUSTER_ADMIN_PASSWORD=$(cat $ROSA_CONFIG | yq -r '.data.admin_password')
         oc login -u $OCP_USERNAME -p $ROSA_CLUSTER_ADMIN_PASSWORD $OCP_SERVER --insecure-skip-tls-verify
-
+        OC_RC=$?
+        if [ $OC_RC -ne 0 ]; then
+          echo "oc login failed, exiting with error.."
+          exit 1
+        fi
```
Comment (whitfiea) [inline tekton/src/tasks/gitops/gitops-deprovision-suite-workspace.yml.j2:73]: I think we have to be careful here. The cluster might not exist but we still want to carry on and remove the pieces from github. This then links to the other piece of work which is to allow deprovsion to run so we need to timeout the argo sync check. So I think we should log the error here but carry on.

**Review by whitfiea — CHANGES_REQUESTED** [review]

File: tekton/src/tasks/gitops/gitops-deprovision-suite-config.yml.j2 (hunk 1 above)
Comment (padmankosalaram) [reply tekton/src/tasks/gitops/gitops-deprovision-suite-config.yml.j2:76]: added the logic to exit with 0 in deprovision flow

File: tekton/src/tasks/gitops/gitops-deprovision-suite-workspace.yml.j2 (hunk 2 above)
Comment (padmankosalaram) [reply tekton/src/tasks/gitops/gitops-deprovision-suite-workspace.yml.j2:73]: added the logic to exit with 0 in deprovision flow

**Review by whitfiea — APPROVED** [review]

**Pull request merged**